src/polyfills.ts

```diff
@@ -55,6 +55,7 @@
 /***************************************************************************************************
  * Zone JS is required by default for Angular itself.
  */
+import 'aframe';
```

Comment:
Does it work if you `import 'aframe';` in `src/components/vr-scatter-plot/vr-scatter-plot.component.ts` instead?